src/datepicker/datepicker.js

```diff
@@ -346,7 +349,9 @@ angular.module('mgcrea.ngStrap.datepicker',
 					'onBeforeShow',
 					'onShow',
 					'onBeforeHide',
-					'onHide'
+					'onHide',
+					'onInvalid',
```

Comment:
These are the callbacks, following the angularstrap convention of callbacks being prefixed with ```bs-```